src/Math-Tests-Numerical/IntegerTest.extension.st

```diff
@@ -0,0 +1,42 @@
+
+{ #category : #'*Math-Tests-Numerical' }
+IntegerTest >> testFindNK [
+
+	self assert: 6 findNK equals:  #(6 1 6 5 4 2).
+	self assert: 10 findNK equals: #(10 1 10 9 5 2 5 3).
+	self assert: 20 findNK equals: #(20 1 20 19 6 3).	
+	self assert: 21 findNK equals: #(21 1 21 20 7 2 7 5).
+	self assert: 55 findNK equals: #(55 1 55 54 11 2 11 9).
+	self assert: 120 findNK equals: #(120 1 120 119 16 2 16 14 10 3 10 7).
+	self assert: 3003 findNK equals: #(3003 1 3003 3002 78 2 78 76 15 5 15 10 14 6 14 8).
+	self assert: 8966473191018617158916954970192684 findNK equals: #(8966473191018617158916954970192684 1 8966473191018617158916954970192684 8966473191018617158916954970192683 123 45 123 78).
+	self assert: 116682544286207712305570174244760883486876241791210077037133735047856714594324355733933738740204795317223528882568337264611289789138133946725471443924278812277695432803500115699090641248357468388106131543801393801287657125117557432072695477147403395443757359171876874010770591355653882772562301453205472707597435095925666815012707478996454360460481159339802667334477440 findNK equals: #(116682544286207712305570174244760883486876241791210077037133735047856714594324355733933738740204795317223528882568337264611289789138133946725471443924278812277695432803500115699090641248357468388106131543801393801287657125117557432072695477147403395443757359171876874010770591355653882772562301453205472707597435095925666815012707478996454360460481159339802667334477440 1 116682544286207712305570174244760883486876241791210077037133735047856714594324355733933738740204795317223528882568337264611289789138133946725471443924278812277695432803500115699090641248357468388106131543801393801287657125117557432072695477147403395443757359171876874010770591355653882772562301453205472707597435095925666815012707478996454360460481159339802667334477440 116682544286207712305570174244760883486876241791210077037133735047856714594324355733933738740204795317223528882568337264611289789138133946725471443924278812277695432803500115699090641248357468388106131543801393801287657125117557432072695477147403395443757359171876874010770591355653882772562301453205472707597435095925666815012707478996454360460481159339802667334477439 1234 567 1234 667).
+	self should: [ 1 findNK ] raise: Error.
+	self should: [ 0 findNK ] raise: Error.
+]
+
+{ #category : #'*Math-Tests-Numerical' }
+IntegerTest >> testFirstOver [
+
+	self assert: (2 firstOver: 3) equals: 2.
+	self assert: (2 firstOver: 4) equals: 2.
+	self assert: (2 firstOver: 5) equals: 3.
+	self assert: (2 firstOver: 6) equals: 3.
+	
+	self assert: (3 firstOver: 3) equals: 2.
+	self assert: (3 firstOver: 4) equals: 2.
+	self assert: (3 firstOver: 5) equals: 2.
+	self assert: (3 firstOver: 6) equals: 2.
+
+	self assert: (4 firstOver: 2) equals: 2.
+	self assert: (4 firstOver: 3) equals: 2.
+	self assert: (4 firstOver: 4) equals: 2.
+	self assert: (4 firstOver: 5) equals: 2.
+	
+	self assert: (4 firstOver: SmallInteger maxVal) equals: 32768.
```

Comment:
SmallInteger maxVal will vary in 32bits vs 64bits image flavours...
Please use an explicit value

Reply:
Done, thanks!